File: Treasury.Domain/Model/ClubClassification.cs
using System.Text.Json.Serialization;

#nullable disable

namespace Treasury.Domain.Model
{
    public partial class ClubClassification
    {
        public string Name { get; set; }
        public string Category { get; set; }

        [JsonIgnore]
        public virtual Organization NameNavigation { get; set; }
    }
}

File: src/main/java/edu/wpi/sga/treasury/domain/model/ClubClassification.java
package edu.wpi.sga.treasury.domain.model;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Getter
@Setter
@Entity
@Table(name = "club_classification")
public class ClubClassification {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Integer id;

    @ManyToOne(fetch = FetchType.LAZY, optional = false)
    @JoinColumn(name = "organization_id", nullable = false)
    private Organization organization;

    @Column(name = "category", nullable = false)
    private String category;

}